spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/actuator/tracing.adoc

```diff
@@ -150,6 +150,7 @@ Tracing with OpenTelemetry and reporting using OTLP requires the following depen
 
 Use the `management.otlp.tracing.*` configuration properties to configure reporting using OTLP.
 
+NOTE: If you need to apply advanced customizations to OTLP span exporters, consider registering a javadoc:org.springframework.boot.actuate.autoconfigure.tracing.otlp.OtlpHttpSpanExporterBuilderCustomizer[] or javadoc:org.springframework.boot.actuate.autoconfigure.tracing.otlp.OtlpGrpcSpanExporterBuilderCustomizer[] beans. These will be invoked prior to the creation of the `OtlpHttpSpanExporter` and `OtlpGrpcSpanExporter`. The customizers take precedence over anything applied by the auto-configuration.
```

Comment:
Should be either:

"consider registering a [...] bean"

or 

"consider registering [...] beans"